carbon-projects/schemas/projectContent.ts

```diff
@@ -76,50 +76,21 @@ export default defineType({
       name: "coverImage",
       description: "Primary cover image to be shown on project page",
       group: "media",
-      type: "image",
-      fields: [
-        {
-          name: "caption",
-          description:
-            "English language caption to show below the image. Can include image attribution if needed.",
-          type: "string",
-        },
-      ],
+      type: "captionImage",
```

Comment:
Will this not break previous coverImages

Reply:
No, these existing images "just work" because the structure of the underlying data is the same

The only thing that breaks is the array of images - we need to recreate each one as a CaptionImage to get the schema to pick them up